### PR TITLE
changed: adjust doxyfile for reuse in applications

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

File: Doxyfile.in [hunk 1]
```diff
@@ -564,10 +564,13 @@ WARN_LOGFILE           =
 # directories like "/usr/src/myproject". Separate the files or directories 
 # with spaces.
 
-INPUT                  = doc src \
-			 src/SIM src/ASM src/ASM/LR \
-			 src/Eig src/LinAlg src/Utility \
-			 @PROJECT_BINARY_DIR@
+INPUT                  = @IFEM_PATH@/src \
+			 @IFEM_PATH@/src/SIM @IFEM_PATH@/src/ASM
```
Comment (kmokstad) [inline Doxyfile.in]: Manglende fortsettelsestegn "\" på linnje 568 Dessuten var indenteringen litt varierende på linjene under.

**Comment by akva2** [comment]
untab'ified, og la til manglende \

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-poisson=7 ifem-elasticity=11 ifem-beamex=4 ifem-advectiondiffusion=7 ifem-darcy=5 ifem-openfrac)18 ifem-fsi=9 ifem-navierstokes=48 ifem-poroelasticity=12 ifem-thermoelasticity=8 ifem-stokes=52 please

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-poisson=7 ifem-elasticity=11 ifem-beamex=4 ifem-advectiondiffusion=7 ifem-darcy=5 ifem-openfrac=18 ifem-fsi=9 ifem-navierstokes=48 ifem-poroelasticity=12 ifem-thermoelasticity=8 ifem-stokes=52 ifem-finitedeformation=4 please

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-poisson=7 ifem-elasticity=11 ifem-beamex=4 ifem-advectiondiffusion=7 ifem-darcy=5 ifem-openfrac=18 ifem-fsi=9 ifem-navierstokes=48 ifem-poroelasticity=12 ifem-thermoelasticity=8 ifem-stokes=52 ifem-finitedeformation=4 please

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-poisson=7 ifem-elasticity=11 ifem-beamex=4 ifem-advectiondiffusion=7 ifem-darcy=5 ifem-openfrac=18 ifem-fsi=9 ifem-navierstokes=48 ifem-poroelasticity=12 ifem-thermoelasticity=8 ifem-stokes=52 ifem-finitedeformation=4 please

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-poisson=7 ifem-elasticity=11 ifem-beamex=4 ifem-advectiondiffusion=7 ifem-darcy=5 ifem-openfrac=18 ifem-fsi=9 ifem-navierstokes=48 ifem-poroelasticity=12 ifem-thermoelasticity=8 ifem-stokes=52 ifem-finitedeformation=4 please

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-poisson=7 ifem-elasticity=11 ifem-beamex=4 ifem-advectiondiffusion=7 ifem-darcy=5 ifem-openfrac=18 ifem-fsi=9 ifem-navierstokes=48 ifem-poroelasticity=12 ifem-thermoelasticity=8 ifem-stokes=52 ifem-finitedeformation=4 please

**Comment by akva2** [comment]
all good. i doubt anyone will review this, so i'll merge.

**Pull request merged**